changelog/1703.breaking.rst

```diff
@@ -0,0 +1,5 @@
+Moved ``plasmapy.formulary.collisions.fundamental_electron_collision_freq`` to
```

Comment:
I would say "deprecates .... and replaces it with ...." since the function isn't just being directly moved but rather replaced.  Also, reference the specific name of the attribute that is replacing it since we will probably be changing the name.